src/topicButton.js

```diff
@@ -32,7 +46,7 @@ var TopicButton = GObject.registerClass({
             hexpand: true,
             label: this._title,
         });
-
+        box.pack_start(this._attentionSign, false, false, 0);
```

Comment:
I might add the attention sign as an overlay child of `overlay` instead of a child of `box`. It looks like it'll be difficult to get it aligned as in the spec if it has to be within the GtkBox's padding.

Reply:
I tried that but overlay can only have one child. Should I try adding it as a child of the frame and see if i can get it to align correctly that way?

Reply:
IIRC, overlays can have one main child but any number of overlay children. Overlay children are added with `add_overlay()` instead of `add()`.